docs/bagit/readme.txt

```diff
@@ -0,0 +1,51 @@
+This zip file contains a HydroShare resource in Bagit format.  
```

Comment:
If it's easy enough to just add it quickly, you might as well.  The comment should be:

# This readme file 

Or something like that.